extension/httpfs/test/test_files/s3_session_token.test

```diff
@@ -0,0 +1,50 @@
+-DATASET CSV empty
+
+--
+
+-DEFINE_STATEMENT_BLOCK AWS_S3_TEST [
+-STATEMENT load from 's3://kuzu-dataset-us/user.csv' return *;
+---- 4
+Adam|30
+Karissa|40
+Noura|25
+Zhang|50
+-STATEMENT CALL s3_region='eu-central-1'
+---- ok
+-STATEMENT CALL s3_url_style='vhost'
+---- ok
+-STATEMENT CREATE NODE TABLE City(name STRING, population INT64, PRIMARY KEY (name))
+---- ok
+-STATEMENT copy City from 's3://kuzu-dataset-eu/city.csv';
+---- ok
+-STATEMENT match (c:City) return c.name, c.population;
+---- 3
+Guelph|22
+Kitchener|200000
+Waterloo|150000
+]
+
+-CASE S3SessionTokenExplicitSet
+-SKIP
```

Comment:
We are skipping this test and S3SessionTokenEnv test globally. Is this expected?

Reply:
yes, the keys and session tokens are only valid for a maximum of 60 minutes.